src/components/Form/Form.tsx

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable */
+import React, { FC, useState } from "react";
+import { Formik, Field } from "formik";
+import {
+  Button,
+  LinearProgress,
+  MenuItem,
+  Snackbar,
+  FormControl,
+  InputLabel,
+} from "@material-ui/core";
+import { TextField, Select } from "formik-material-ui";
+import * as yup from "yup";
+import FormContainer from "components/FormContainer/FormContainer";
+import MuiAlert, { AlertProps } from "@material-ui/lab/Alert";
```

Comment:
@mlh-dungpq cc: @mlh-binhnt @themitvp 
`@material-ui/lab ` still in alpha version. is this often used in ML's production projects?
https://www.npmjs.com/package/@material-ui/lab
![image](https://user-images.githubusercontent.com/16083322/103508817-18592d80-4ea5-11eb-9fd1-404146ee260c.png)